### PR TITLE
Unittest

### PR DESCRIPTION
**Pull request by ng-anhhtuann**
Edit in some other API
Write unittest adminRight

### CODE REVIEW

File: src/test/java/com/example/GHTK/ShopTest/ShopPutTest.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package com.example.GHTK.ShopTest;
```
Comment (uranashel44) [inline src/test/java/com/example/GHTK/ShopTest/ShopPutTest.java:1]: Any logic?

File: src/test/java/com/example/GHTK/ShopTest/ShopPostTest.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package com.example.GHTK.ShopTest;
+
```
Comment (uranashel44) [inline src/test/java/com/example/GHTK/ShopTest/ShopPostTest.java:2]: Same?

File: src/test/java/com/example/GHTK/ShopTest/ShopGetTest.java [hunk 3]
```diff
@@ -0,0 +1,4 @@
+package com.example.GHTK.ShopTest;
+
+public class ShopGetTest {
```
Comment (uranashel44) [inline src/test/java/com/example/GHTK/ShopTest/ShopGetTest.java:3]: Same?

**Review by uranashel44 — APPROVED** [review]
LGTM but some functions still missing

**Comment by ng-anhhtuann** [comment]
> LGTM but some functions still missing

I still have some things wrong with these class

**Pull request merged**